epub33/fxl-a11y/index.html

```diff
@@ -67,36 +67,19 @@ <h3>Overview</h3>
 
             <p>The main motivation behind creating fixed layout publications is the need to preserve the print layout of the book, either because of it's importance to the text (i.e. complex diagrams) or it's artistic purpose (i.e. illustrated text). However, these publications are often partially or completely inaccessible to people with print disabilities.</p>
 
-            <p>This note serves to help content authors and publishers try to address some of the common accessibility issues found in fixed layout content, including navigation, reading order, and text alternatives. This document is a companion to [[epub-a11y-11]], specifically for fixed layout publications. All recommendations made in [[epub-a11y-11]], [[epub-33]], and [[epub-rs-33]] are applied and extended here.</p>
+            <p>This note serves to help content authors, publishers, and reading systems try to address some of the common accessibility issues found in fixed layout content including navigation, reading order, and text alternatives. This document is a companion to [[epub-a11y-11]], specifically for fixed layout publications. All recommendations made in [[epub-a11y-11]], [[epub-33]], and [[epub-rs-33]] are applied and extended here.</p>
         </section>
 
         <section id="intro-limits">
             <h3>The limits of fixed layout accessibility</h3>
```

Comment:
PDFs can be WCAG compliant, I think we should analyze WCAG 2.2 with this perspective and check if all Success Criteria applicable to PDFs are applicabile to EPUBs FXL